src/pages/Beta/Pool/DetailModal/DetailView/index.tsx

```diff
@@ -27,14 +27,17 @@ const DetailView = ({ stakingInfo, onDismiss, version }: PoolDetailProps) => {
   const currency1 = unwrappedToken(token1, chainId)
 
   const isStaking = Boolean(stakingInfo?.stakedAmount?.greaterThan('0'))
+  const [, pair] = usePair(token0, token1)
+  const { userPgl, liquidityInUSD } = useGetPoolDollerWorth(pair)
 
+  const hasliquidity = isStaking || userPgl?.greaterThan('0') || liquidityInUSD > 0
   return (
     <>
       <MobileWrapper>
         <Header stakingInfo={stakingInfo} onClose={onDismiss} />
         <Box p={10}>
-          {isStaking && <EarnDetail stakingInfo={stakingInfo} version={version} />}
-          <Box mt={isStaking ? '10px' : '0px'}>
+          {hasliquidity && <EarnDetail stakingInfo={stakingInfo} version={version} />}
+          <Box mt={hasliquidity ? '10px' : '0px'}>
```

Comment:
EarnDetail is only for users who are farming. By showing EarnDetail earn details to users who just have liquidity will confuse user.